beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -117,6 +118,10 @@ func (vs *Server) getPayloadHeader(ctx context.Context, slot types.Slot, idx typ
 	if err != nil {
 		return nil, err
 	}
+	if err := vs.validateBuilderSignature(bid); err != nil {
```

Comment:
How does this work if we dont run with a builder ?

Reply:
`getAndBuildHeaderBlock` and `getPayloadHeader` won't get called because`vs.BlockBuilder.Configured()` will return false. I also can be super defensive and check `vs.BlockBuilder.Configured()` again before `GetHeader` and return an error if it's false. (That's something you will like?)